### ISSUE TITLE
Is there any good way to get account related replies with steemjs?

### ISSUE THREAD

**Issue by Cha0s0000** [steemit/steem-js#367]
instead of using get_account_history , is there any better way ?

**Comment by BartolomeoItaliano** [comment]
Hi, 
Yep it is:

`steem.api.getRepliesByLastUpdate(startAuthor, startPermlink, limit, callback);
`
as startPermlink pass empty string and you will get them.

**Comment by Cha0s0000** [comment]
@BartolomeoItaliano Thank you . btw , i found that there is an another way which using the 
`https://api.steemjs.com/get_state?path=/@' + author + '/recent-replies`  
[then filter the object .](https://github.com/steemit/steem-python/search?utf8=%E2%9C%93&q=recent-replies&type=)
this is different from 
`https://api.steemjs.com/get_state?path=/@' + author `

**Comment by BartolomeoItaliano** [comment]
Yes of course steem-js also used similar endpoints, you choose it by setting config. 
Steem-js wraps these endpoints to be more programmer friendly :)

**Comment by ety001** [comment]
This repository has been superseded by a complete 2025 rewrite. The new trunk (`master`, starting at v1.1.0) is a ground-up rewrite of `@steemit/steem-js` and the legacy 0.7.x / 0.8.x code line is archived.

As part of that transition, this issue is being closed because:
- It was filed against the legacy codebase, and the code it references no longer exists in the rewritten trunk.
- The new codebase reimplements the affected areas (RPC routing, transaction serialization, auth/signing, crypto, browser builds) from scratch, so this issue cannot be carried over as-is.

If the problem or request still applies to the rewritten codebase (v1.1.0+ on `master`), please open a new issue against the current trunk with details that reproduce on the new version. Thank you for taking the time to report this originally.
